src/components/Modals/NewMessageModal.jsx

```diff
@@ -32,11 +32,12 @@ import { MessageContext, SignedInUserContext } from '@contexts';
  * @param {boolean} Props.showModal - Boolean for showing message modal
  * @param {React.Dispatch<React.SetStateAction<boolean>>} Props.setShowModal
  * - React set function for showModal
+ * @param Props.toField
```

Comment:
The JSDoc for Line 35 needs to be corrected. It's missing the type and a description. The following format should do:

```js
/*
 * @param {variable_type} variable_in_question - Description of variable
 */
```

You could check the other `@param` lines in the JSDoc for reference.

Reply:
All the changes should be implemented now sans the `sendMessageTTL`. Still working on that.

Also wanted to mention that going to the Messages tab gives this error:
```
MessagePreview.jsx:100 Warning: Each child in a list should have a unique "key" prop.
```

Reply:
> All the changes should be implemented now sans the `sendMessageTTL`. Still working on that.
> 
> Also wanted to mention that going to the Messages tab gives this error:
> 
> ```
> MessagePreview.jsx:100 Warning: Each child in a list should have a unique "key" prop.
> ```

This error is related to the rendering of the message list. @bingeboy got that fixed in PR #460. We can ignore that here.